### ISSUE TITLE
Fix typedoc generation

### ISSUE THREAD

**Issue by s1hofmann** [sakuli/sakuli#29]
Typedoc generation fails on Travis and causes the master build to fail:

```
7.21s$ if [[ "$TRAVIS_BRANCH" == "master" ]]; then lerna run typedoc && gh-pages -d docs; fi
lerna notice cli v3.4.3
lerna info Executing command in 5 packages: "npm run typedoc"
lerna ERR! npm run typedoc exited 4 in '@sakuli/commons'
lerna ERR! npm run typedoc stdout:
> @sakuli/commons@2.0.0-RC1 typedoc /home/travis/build/sakuli/sakuli/packages/sakuli-commons
> typedoc --out ../../docs/api/sakuli-cli --hideGenerator --mode file --excludeExternals --readme README.md --exclude *.spec.* src/
Using TypeScript 3.1.6 from /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/typedoc/node_modules/typescript/lib
Error: /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/memfs/lib/Stats.d.ts(1)
 Cannot find name 'bigint'.
Error: /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/memfs/lib/Stats.d.ts(7)
 Cannot find name 'bigint'.
Error: /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/memfs/lib/volume.d.ts(226)
 Cannot find name 'bigint'.
Error: /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/memfs/lib/volume.d.ts(236)
 Cannot find name 'bigint'.
Error: /home/travis/build/sakuli/sakuli/packages/sakuli-commons/node_modules/memfs/lib/volume.d.ts(246)
 Cannot find name 'bigint'.
lerna ERR! npm run typedoc stderr:
npm ERR! code ELIFECYCLE
npm ERR! errno 4
npm ERR! @sakuli/commons@2.0.0-RC1 typedoc: `typedoc --out ../../docs/api/sakuli-cli --hideGenerator --mode file --excludeExternals --readme README.md --exclude *.spec.* src/`
npm ERR! Exit status 4
npm ERR! 
npm ERR! Failed at the @sakuli/commons@2.0.0-RC1 typedoc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-04-12T06_35_36_713Z-debug.log
lerna ERR! npm run typedoc exited 4 in '@sakuli/commons'
The command "if [[ "$TRAVIS_BRANCH" == "master" ]]; then lerna run typedoc && gh-pages -d docs; fi" exited with 4.
```

**Comment by tnobody** [comment]
Could be solved with #45

**Comment by s1hofmann** [comment]
Closed by #53
